src/common/options.cc

```diff
@@ -2950,13 +2950,10 @@ std::vector<Option> get_global_options() {
 
     Option("osd_op_queue", Option::TYPE_STR, Option::LEVEL_ADVANCED)
     .set_default("wpq")
-    .set_enum_allowed( { "wpq", "prioritized",
-	  "mclock_opclass", "mclock_client", "mclock_scheduler",
-	  "debug_random" } )
+    .set_enum_allowed( { "wpq", "mclock_scheduler", "debug_random" } )
```

Comment:
since you are at here, could you drop all occurrences of "mclock_client" and "mclock_opclass"?

i think "mclock_scheduler" is still used by crimson. could you keep it?

Reply:
i can drop the related, but there need to  update the section "QoS Based on mClock" in osd-config-ref.rst, i don't know the implement detials.